client/java/client-java-configurations.md

```diff
@@ -0,0 +1,25 @@
+---
+title: Java Client 配置参数
+aliases: ['/docs-cn/dev/client/java/client-java-configuration/','/docs-cn/dev/reference/client/java/client-java-configuration/']
+---
+
+# Java Client 配置参数
+
+本文介绍了与部署使用 Java Client 相关的配置参数。
```

Comment:
Could you give more description, such as how Java Client relates to TiDB or other components, what are the user scenarios and benefits of using Java Client, what to pass in the following configuration, what are the restrictions?

Reply:
Enclosed a new document about how to use Java Client. Some documents about API usage will be added later because I think they should not be included in this PR.

Reply:
> how to pass in the following configuration

It is described in the next session. The configurations will be passed through JVM parameters.